cpg_workflows/utils.py

```diff
@@ -15,44 +15,61 @@
 from random import choices
 from typing import Union, cast
 
+import coloredlogs
+
 import hail as hl
 from hailtop.batch import ResourceFile
 
 from cpg_utils import Path, to_path
 from cpg_utils.config import config_retrieve, get_config
 
-LOGGER: logging.Logger | None = None
+DEFAULT_LOG_FORMAT = '%(asctime)s - %(name)s - %(pathname)s: %(lineno)d - %(levelname)s - %(message)s'
```

Comment:
Do you still want `asctime` in the default logger, since that was one of the issues in the PR description?

Reply:
I think yes - in general usage it's really valuable to determine how long different parts of the process are taking. It's a kind of soft-profiling we get for free using timestamps.

The adaptation in this change is that if I'm running something to be equivalent plain text for comparison purposes I would add a config entry `workflow.logger.NAME.format` and set that to be just "%(message)s". That would override the default on a per-logger basis.

I've just pushed a more general change so that we can override this default for all loggers with a single config entry.